nucypher/characters/chaotic.py

```diff
@@ -142,7 +142,7 @@ class Felix(Character, NucypherTokenActor):
     BATCH_SIZE = 10                 # transactions
     MULTIPLIER = 0.95               # 5% reduction of previous stake is 0.95, for example
     MINIMUM_DISBURSEMENT = 1e18     # NuNits
-    # TRANSACTION_GAS = 40000       # gas  TODO
+    ETHER_AIRDROP_AMOUNT = int(2e18)  # Wei
```

Comment:
Should these values be a constant/envvar?

Reply:
Hmm, I actually like it right here on `Felix.ETHER_AIRDROP_AMOUNT`.

Reply:
Follow-up Issue https://github.com/nucypher/nucypher/issues/1050